assets/js/blocks/cart/inner-blocks/filled-cart-block/frontend.tsx

```diff
@@ -24,7 +24,7 @@ const FrontendBlock = ( {
 	const { hasDarkControls } = useCartBlockContext();
 	const { createErrorNotice } = useDispatch( 'core/notices' );
 
-	// Ensures any cart errors listed in the API response get shown.
+	// @todo Cart errors need to be watched for and created as notices elsewhere.
```

Comment:
This requires a follow up so we can dispatch cart errors on both cart and checkout using the same logic.